.github/workflows/build.yml

```diff
@@ -1,71 +1,113 @@
 name: 'build'
-
 on:
-  push:
-    branches:
-      - 'main'
-    tags:
-      - 'v*'
-    workflow_dispatch:
-      inputs:
-        tag:
-          description: 'Tag to apply to the Docker image. If unspecified, defaults to the latest SHA.'
-          type: 'string'
-          default: ''
-
-concurrency:
-  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
-  cancel-in-progress: true
-
+  workflow_call:
+    inputs:
+      env-context:
+        required: true
+        type: string
+        description: 'Shared environment data from the calling workflow'
```

Comment:
Yes, its a json string. You can see the creation of it in the ci.yml file.

It looks something like:

`"{\"docker_registry\":\"us-docker.pkg.dev\",\"docker_tag\":\"7e8307e8af56c601e205591c32ac4ad5a1ff9809\",\"docker_repo\":\"us-docker.pkg.dev/lumberjack-ci-dev/images\",\"wif_provider\":\"projects/638387980668/locations/global/workloadIdentityPools/github-pool/providers/github-provider\",\"wif_service_account\":\"gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com\",\"server_project_id\":\"github-ci-server\",\"client_project_id\":\"github-ci-app-0\",\"tf_state_bucket\":\"lumberjack-ci-tfstate\",\"bigquery_dataset_id\":\"audit_logs.audit_abcxyz_data_access\"}"`

Reply:
🤦  - if I'm getting rid of 'ci.yml' in favor of chaining them together the other way then this is irrelevant since we won't have centralized configuration that we need to pass around anymore and each workflow can just hold the parts it needs.